### ISSUE TITLE
Add comment with warning about automatically generated file

### ISSUE THREAD

**Issue by ikokostya** [bcherny/json-schema-to-typescript#84]
Because typescript interface is generated from JSON schema automatically it would be helpful to add comment about this at the beginning of output file, e.g.:

```
// WARNING: this file generated automatically using [json-schema-to-typescript](https://github.com/bcherny/json-schema-to-typescript)
// tool. Please, don't edit it manually.
```

**Comment by bcherny** [comment]
Published in 4.3.0.

**Comment by lukeautry** [comment]
It would actually be nice to prevent these comments from being added through a setting. Depending on your usage, you may get interfaces being annotated with this warning message that you don't really want sticking around.

**Comment by bcherny** [comment]
@lukeautry Can you tell me a little more about your use case. Is it a one-time conversion, that you don't want to keep in sync?

**Comment by lukeautry** [comment]
Sure - I have a custom json file generated by the server that contains json schemas for a bunch of different interfaces. Using your library, we convert those to TypeScript interfaces that can be used in client side code. Given that we plan to distribute a library with typings shipped and these generated interfaces included, we don't want the autogenerated warning to ship with those.

**Comment by bcherny** [comment]
That sounds like a reasonable request. I'm tracking it in https://github.com/bcherny/json-schema-to-typescript/issues/126 - PRs are welcome!
